### PR TITLE
Adding extra-text to be send with QR code in showQrActivity

### LINKED ISSUE #1131: Feature: Adding extra text to be share with QR code in android sharesheet in showQrActivity
**Summary:**

Adding extra text to be shared with QR code with Intent in android Sharesheet in showQrActivity, for better user experience.

**Steps to reproduce:**

1. Go to Request fragment in Payments 
2. Click on show code
3. Click on the share icon (located at top right)
4. Choose any app to share

**Expected behavior:**

The extra text should be share with a QR code to ease the user experience

**Device and Android version:**

Redmi Note 5 Pro
Android 9 Pie
custom ROM

### PR DESCRIPTION
**Pull request by Harshad09**
## Issue Fix
Fixes #1131

## Screenshots

WhatsApp Image 2021-01-07 at 12 49 46 AM (1)
WhatsApp Image 2021-01-07 at 12 49 46 AM
WhatsApp Image 2021-01-07 at 1 09 29 AM

## Description
Added Intent.EXTRA_STRING to add a text that should be sent along with the QR code when we share the code to different apps using the android sharesheet.
##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Harshad09** [comment]
@devansh-299  @s-ayush2903 Can you Please Review my PR

File: mifospay/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -292,6 +292,7 @@
     <string name="enter_VPA">Please enter the VPA</string>
     <string name="amount_header">Amount:</string>
     <string name="deleted_successfully">Deleted Successfully</string>
+    <string name="scan_mifos_qr">Scan this QR code to pay through Mifos Pay</string>
```
Comment (s-ayush2903) [inline mifospay/src/main/res/values/strings.xml]: Its _MifosPay_ not _Mifos Pay_

**Comment by Harshad09** [comment]
@s-ayush2903  Changed _Mifos Pay_ to _MifosPay_

**Comment by s-ayush2903** [comment]
LGTM

**Comment by Prashant830** [comment]
@Harshad09 I think, In your branch have a code conflict . Are u still able to resolved it??

**Comment by Prashant830** [comment]
@Harshad09 I think you are not able to solve this conflict code at this time. So now  I am going to create new pr for this issue.

**Pull request closed without merging**